two_phase_client_humble:
  ros__parameters:
    frame_id: "map"
    goal_a:
      - 5.7
      - 8.75
      - 1.57
    goal_b:
      - 5.67
      - 9.6
      - 1.57